bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -113,34 +112,112 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
+            
+            Bugzilla
+            bugzilla.mozilla.org (Do not create bugs or comment on existing bugs while testing)
+            
+            Crash report (socorro)
+            crash-stats.mozilla.com
+            
+            Firefox Accounts
+            accounts.firefox.com
+            api.accounts.firefox.com
+            oauth.accounts.firefox.com
+            profile.accounts.firefox.com
+            verifier.accounts.firefox.com
+            
+            Hello
+            hello.firefox.com
+            loop.services.mozilla.com
+            msisdn.services.mozilla.com
+            
+            Input
+            input.mozilla.org
+            
+            Locations
+            location.services.mozilla.com
+            
+            Marketplace
+            marketplace.firefox.com
+            receiptcheck.marketplace.firefox.com
+            static.marketplace.firefox.com
+            
+            Mercurial
+            hg.mozilla.org
+            
+            Metrics
+            metrics.services.mozilla.com
+            
+            MozTrap
+            moztrap.mozilla.org
+            
+            Phonebook
+            phonebook.mozilla.org
+            
+            Product Delivery
+            download.mozilla.org
+            download-installer.cdn.mozilla.net
+            
+            Push
+            push.services.mozilla.com
+            push1.push.hello.firefox.com
+            updates.push.services.mozilla.com
+            
+            SSO
+            sso.mozilla.com (*not* mozilla.okta.com)
+            ldap.mozilla.org
+            
+            SUMO
+            support.mozilla.org
+            
+            Sync
+            token.services.mozilla.com
+            sync-264-us-west-2.sync.services.mozilla.com
+            
+            Teach
+            teach.mozilla.org
+            
+            Tiles
+            tiles.cdn.mozilla.net
+            tiles.services.mozilla.com
+            
+            TLS Observatory
+            tls-observatory.services.mozilla.com
+            
+            Tracking Protection
+            shavar.services.mozilla.com
+            tracking.services.mozilla.com
+          </pre>
 
-          <p>The Mozilla Developer Network (developer.mozilla.org) has been removed from this
-            list due to site vandalism by researchers looking for security issues. It will be
+          <p>The Mozilla Developer Network (developer.mozilla.org) and wiki.mozilla.org have  been removed from this
+            list due to site vandalism by researchers looking for security issues. They will be
             added back at a later date when a test server is deployed to avoid negative impacts
             on our primary reference site.</p>
 
-          <p>Only Mozilla websites and services are in scope, not email phishing attacks
+          <p>Only Mozilla websites and services are in scope, not denial of service attacks, email phishing attacks
```

Comment:
Is this intended to exclude all DoS attacks, including ones that are application-specific? If so this is an odd place to say it.